src/egl/drivers/dri2/egl_dri2.c

```diff
@@ -944,6 +944,11 @@ dri2_initialize(_EGLDriver *drv, _EGLDisplay *disp)
    case _EGL_PLATFORM_ANDROID:
       ret = dri2_initialize_android(drv, disp);
       break;
+#ifdef HAVE_YUNOS_PLATFORM
+   case _EGL_PLATFORM_YUNOS:
+      ret = dri2_initialize_yunos(drv, disp);
```

Comment:
Did you forget to include the implementation in this PR?

Reply:
The implementation is in YUNOS(AliOS) platform egl driver. 

All i965 relative changes have been removed. The main changes of this PR are yunos.mk and platform_yunos.c. Both are new added and for YunOS(AliOS) only. they won't introduce rebase conflict.